contracts/extensions/CoinMachine.sol

```diff
@@ -231,7 +238,7 @@ contract CoinMachine is DSMath, ColonyExtension {
   /// @notice Get the number of remaining tokens for sale this period
   function getNumAvailable() public view returns (uint256) {
```

Comment:
This function goes wrong if once we get close to the maximum (i.e. if `maxPerPeriod` is more than the total that can be sold due to hitting `maxTotal`).